docs/content/operations/insert-segment-to-db.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: doc_page
+---
+# insert-segment-to-db Tool
+
+`insert-segment-to-db` is a tool that can insert segments into Druid metadata storage. It is intended to be used
```

Comment:
this doc is not linked anywhere, how are users expected to find it? can you add it to "operations" section in toc file?